README.md

```diff
@@ -170,8 +170,8 @@ The data to import should be structured like this:
 
 ## :bulb: Importing to a space with existing content
 
-- Both source space and destination space must share the same content model structure. In order to achieve that, please use the [migration-cli](https://www.npmjs.com/package/contentful-migration-cli).
-- Content transformations are also not supported, please use the [migration-cli](https://www.npmjs.com/package/contentful-migration-cli).
+- Both source space and destination space must share the same content model structure. In order to achieve that, please use the [cli](https://www.npmjs.com/package/contentful-cli).
+- Content transformations are also not supported, please use the [cli](https://www.npmjs.com/package/contentful-cli).
```

Comment:
Yes, they use the regular CLI, but they have to use our migration tooling.

"contentful space migration" -- maybe we should try and be clear here? or link to the contentful-migration github repo?

Reply:
Thanks, that makes sense! I will think about a clearer explanation.